source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -317,6 +349,34 @@ TEST_F(NiDAQmxDriverApiTests, AOVoltageChannel_WriteAODataWithOutOfRangeValue_Re
 
   EXPECT_EQ(DAQmxErrorInvalidAODataWrite, write_response.status());
 }
+
+TEST_F(NiDAQmxDriverApiTests, TaskWithAOChannel_GetNthTaskDevice_ReturnsDeviceForChannel)
+{
+  const double AO_MIN = 1.0;
+  const double AO_MAX = 10.0;
+  CreateAOVoltageChanResponse create_channel_response;
+  create_ao_voltage_chan(AO_MIN, AO_MAX, create_channel_response);
```

Comment:
You can simplify this to just be `create_ao_voltage_chan(AO_MIN, AO_MAX)` since you introduced the default throwaway response option for it.